### PR TITLE
Update monitor-with-resource-health.md

### PR DESCRIPTION
**Pull request by elibery**
@orspod Added link to "System information" page in case of Unavailable health status to help users debug the problem by themselves.

### CODE REVIEW

**Comment by PRMerger8** [comment]
@elibery : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/monitor-with-resource-health.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ A health status of **Unavailable** indicates that there's an ongoing problem wit
 
 ![Unavailable](media/monitor-with-resource-health/unavailable.png)
 
+You can use following [commands](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/management/systeminfo) to understand the source of the problem by yourself.
```
Comment (orspod) [inline data-explorer/monitor-with-resource-health.md:46]:
```suggestion
> [!TIP]
> You can use the [system information commands](kusto/management/systeminfo.md) to find the source of the problem.
```

**Comment by PRMerger9** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by orspod** [comment]
@elibery - thanks for your contributions.
1. You should be using the dataexplorer-docs-pr repo for docs contributions. See the setup process in our internal docs.
2. Links should be relative (since within same repo).
3. I made some changes and merged the PR.

**Pull request merged**